test/conftest.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+from fastapi.testclient import TestClient
+from main import app
+
+
+@pytest.fixture
+def client():
+    return TestClient(app)
```

Comment:
Are you sure this executes startup and shutdown events? Why did you drop context manager?